joomla/joomla-cms
# [4] com_csp empty state

### Summary of Changes

Add empty state for System -> Content Security Policy 

### Testing Instructions

If you want a NON empty state you can insert this to your db to see the NON empty state.

```sql
INSERT INTO `jos_csp` (`id`, `document_uri`, `blocked_uri`, `directive`, `client`, `created`, `modified`, `published`)
VALUES
	(1, '/from', '', '', '', '1978-01-01 11:11:11', '1978-01-01 11:11:11', 0);
```
### Actual result BEFORE applying this Pull Request

<img width="1599" alt="Screenshot 2021-04-29 at 21 51 01" src="https://user-images.githubusercontent.com/400092/116616798-01978480-a935-11eb-99d8-a7407833bc16.png">


### Expected result AFTER applying this Pull Request

<img width="1596" alt="Screenshot 2021-04-29 at 21 47 19" src="https://user-images.githubusercontent.com/400092/116616733-f0e70e80-a934-11eb-822a-c808b5e14a62.png">


### Documentation Changes Required
